Sources/Classes/SMStore.swift

```diff
@@ -296,7 +296,16 @@ open class SMStore: NSIncrementalStore {
         moc.persistentStoreCoordinator = self.backingPersistentStoreCoordinator
         moc.retainsRegisteredObjects = true
         moc.mergePolicy = NSMergeByPropertyObjectTrumpMergePolicy
-        return moc     }()
+        return moc
+    }()
+    
+    fileprivate lazy var localStoreMOC: NSManagedObjectContext = {
+        var moc = NSManagedObjectContext(concurrencyType: NSManagedObjectContextConcurrencyType.privateQueueConcurrencyType)
+        moc.persistentStoreCoordinator = self.persistentStoreCoordinator
+        moc.retainsRegisteredObjects = true
+        moc.mergePolicy = NSMergeByPropertyObjectTrumpMergePolicy
+        return moc
+    }()
```

Comment:
Does separating out this `localStoreMOC` solve #96?

Reply:
I changes I made don't solve this. I am not sure what the specific cause of this annoying warning is, but from my looking through the code it seemed to be ok, so it may be a false positive - a needless warning.